code/lstm.py

```diff
@@ -428,26 +377,30 @@ def pred_error(f_pred, prepare_data, data, iterator, verbose=False):
     return valid_err
 
 
-def train(dim_proj=100,
-          patience=10,
-          max_epochs=5000,
-          dispFreq=100,
-          activ='lambda x: tensor.tanh(x)',
-          decay_c=0.,
-          lrate=0.01,
-          n_words=100000,
-          data_sym=False,
-          optimizer='rmsprop',
-          encoder='rconv',
-          saveto='model.npz',
-          noise_std=0.,
-          validFreq=1000,
-          saveFreq=1000,  # save the parameters after every saveFreq updates
-          maxlen=50,
-          batch_size=16,
-          valid_batch_size=16,
-          dataset='sentiment140',
-          use_dropout=False):
+def test_lstm(
+    dim_proj=128,  # TODO: What is this
```

Comment:
I would guess it's the size of the projection (hidden recurrent representation). We should make sure and put it in the comment.